xml/System.Net.Security/SslClientHelloInfo.xml

```diff
@@ -40,7 +40,7 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The host server specified by the client.</summary>
```

Comment:
I would change that to "The host server specified by the client in Server Name extension (SNI). If the extension is not present this returns empty string.